src/cpp/fastdds/publisher/DataWriter.cpp

```diff
@@ -98,19 +98,68 @@ ReturnCode_t DataWriter::write(
     return impl_->write(data, handle);
 }
 
+ReturnCode_t DataWriter::write_w_timestamp(
+        void* data,
+        const InstanceHandle_t& handle,
+        const fastrtps::rtps::Time_t& timestamp)
+{
+    static_cast<void> (data);
+    static_cast<void> (handle);
+    static_cast<void> (timestamp);
+    return ReturnCode_t::RETCODE_UNSUPPORTED;
+}
+
 InstanceHandle_t DataWriter::register_instance(
         void* instance)
 {
     return impl_->register_instance(instance);
 }
 
+InstanceHandle_t DataWriter::register_instance_w_timestamp(
+        void* instance,
+        const fastrtps::rtps::Time_t& timestamp)
+{
+    static_cast<void> (instance);
+    static_cast<void> (timestamp);
+    logWarning(DataWriter, "register_instance_w_timestamp method not yet implemented")
```

Comment:
```suggestion
    logWarning(DATA_WRITER, "register_instance_w_timestamp method not yet implemented")
```
Usually log tag is used with Capital Letters